File: app/src/main/java/com/acrcloud/data/Album__.java

package com.acrcloud.data;

import android.os.Parcel;
import android.os.Parcelable;
import android.os.Parcelable.Creator;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Album__ implements Parcelable
{

    @SerializedName("name")
    @Expose
    private String name;
    public final static Creator<Album__> CREATOR = new Creator<Album__>() {


        @SuppressWarnings({
            "unchecked"
        })
        public Album__ createFromParcel(Parcel in) {
            return new Album__(in);
        }

        public Album__[] newArray(int size) {
            return (new Album__[size]);
        }

    }
    ;

    protected Album__(Parcel in) {
        this.name = ((String) in.readValue((String.class.getClassLoader())));
    }

    /**
     * No args constructor for use in serialization
     * 
     */
    public Album__() {
    }

    /**
     * 
     * @param name
     */
    public Album__(String name) {
        super();
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void writeToParcel(Parcel dest, int flags) {
        dest.writeValue(name);
    }

    public int describeContents() {
        return  0;
    }

}

File: app/src/main/java/com/acrcloud/data/Album_.kt
package com.acrcloud.data

import android.os.Parcel
import android.os.Parcelable

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Album_ : Parcelable {

    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("id")
    @Expose
    var id: String? = null

    protected constructor(`in`: Parcel) {
        this.name = `in`.readValue(String::class.java.classLoader) as String
        this.id = `in`.readValue(String::class.java.classLoader) as String
    }

    /**
     * No args constructor for use in serialization
     */
    constructor() {}

    /**
     * @param id
     * @param name
     */
    constructor(name: String, id: String) : super() {
        this.name = name
        this.id = id
    }

    override fun writeToParcel(dest: Parcel, flags: Int) {
        dest.writeValue(name)
        dest.writeValue(id)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Album_> {
        override fun createFromParcel(parcel: Parcel): Album_ {
            return Album_(parcel)
        }

        override fun newArray(size: Int): Array<Album_?> {
            return arrayOfNulls(size)
        }
    }


}
